src/rsa.rs

```diff
@@ -295,7 +296,16 @@ impl RSAKeyPair {
                 try!(bssl::map_result(unsafe {
                     rsa_new_end(rsa.as_mut())
                 }));
-                Ok(RSAKeyPair { rsa: rsa })
+                let blinding = unsafe { BN_BLINDING_new() };
+                if blinding.is_null() {
+                    Err(())
+                }
+                else {
```

Comment:
In *ring* we usually use the early-return-on-errors style consistently (if we don't, it's probably not intentional). So, I'd prefer to have it be `return Err(());` with no `else`.